proto/cork/v1/tx.proto

```diff
@@ -22,3 +22,15 @@ message MsgSubmitCorkRequest {
 }
 
 message MsgSubmitCorkResponse {}
+
+// MsgScheduleCorkRequest - sdk.Msg for scheduling a cork request for on or after a specific block height
+message MsgScheduleCorkRequest {
```

Comment:
We're creating a `ScheduledCork` object in `cork.proto` which also contains these fields. We can probably use that in place of the `cork` and `block_height` objects here. It's duplicating the signer but that's probably fine, unless you want to have `MustGetSigner` look into the object.

Reply:
Optionally, could delete the ScheduledCork proto, since it is currently used nowhere.